spec/dummy/app/models/card.rb

```diff
@@ -0,0 +1,34 @@
+class Card < ActiveRecord::Base
+  belongs_to :block
+  belongs_to :user
+  before_validation :set_review_date_as_now, on: :create
+  validates :user_id, presence: true
+  validates :original_text,
+            :translated_text,
+            :review_date,
+            presence: { message: "Необходимо заполнить поле." }
+  validates :user_id, presence: { message: "Ошибка ассоциации." }
+  validates :block_id,
+            presence: { message: "Выберите колоду из выпадающего списка." }
+  validate :texts_are_not_equal
+
+  scope :pending, -> { where("review_date <= ?", Time.now.rfc3339).
```

Comment:
Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.